app/views/layouts/_user_header.html.erb

```diff
@@ -0,0 +1,86 @@
+            <li class="active has-sub">
+                <%= link_to 'add resume', new_resume_path %>
+            </li>
+            <li class="active has-sub">
+                <%= link_to 'show resumes', resumes_path %>
+            </li>
+            <li class="active has-sub">
+                <%= link_to "Available Jobs", jobs_list_user_opening_jobs_path(current_user)%>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </aside>
+    <!-- END MENU SIDEBAR-->
+
+    <!-- PAGE CONTAINER-->
+    <div class="page-container">
+      <!-- HEADER DESKTOP-->
+      <header class="header-desktop">
+        <div class="section__content section__content--p30">
+          <div class="container-fluid">
+            <div class="header-wrap">
+              <form class="form-header" action="" method="POST">
+              </form>
+              <div class="header-button">
+
+                <div class="account-wrap">
+                  <div class="account-item clearfix js-item-menu">
+                    <%= link_to current_user.email, edit_user_registration_path %>
+                    <%= link_to "Logout", destroy_user_session_path, method: :delete,class: "btn btn-outline-danger"%>
+                    <div class="account-dropdown js-dropdown">
+                      <div class="info clearfix">
+                        <div class="image">
+                          <a href="#">
+                            <img src="images/icon/avatar-01.jpg" alt="John Doe" />
+                          </a>
+                        </div>
+                        <div class="content">
+                          <h5 class="name">
+                            <a href="#">john doe</a>
+                          </h5>
+                          <span class="email">johndoe@example.com
+                        </div>
+
+                      </div>
+                      <div class="account-dropdown__body">
+                        <div class="account-dropdown__item">
+                          <a href="#">
+                            <i class="zmdi zmdi-account"></i>Account</a>
+                        </div>
+                        <div class="account-dropdown__item">
+                          <a href="#">
+                            <i class="zmdi zmdi-settings"></i>Setting</a>
+                        </div>
+                        <div class="account-dropdown__item">
+                          <a href="#">
+                            <i class="zmdi zmdi-money-box"></i>Billing</a>
+                        </div>
+                      </div>
+                      <div class="account-dropdown__footer">
+                        <a href="#">
+                          <i class="zmdi zmdi-power"></i>Logout</a>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </header>
+      <!-- HEADER DESKTOP-->
+      
```

Comment:
EOL

Reply:
EOL is already added and if you are asking about incomplete tags then we completed it in another page as we have header and side bar in our head bar.